_posts/0000-01-02-darzu84.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
This will be tricky never done this @darzu84 will learn to code and will be the best ""''# what is going on
we are trying to insert an image also
![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png)